FLT/Mathlib/MeasureTheory/Group/Action.lean

```diff
@@ -0,0 +1,66 @@
+import Mathlib.MeasureTheory.Group.Action
+import Mathlib.MeasureTheory.Group.Pointwise
+import Mathlib.Topology.Algebra.InfiniteSum.ENNReal
+import FLT.Mathlib.Algebra.Group.Subgroup.Defs
+import FLT.Mathlib.GroupTheory.Complement
+import FLT.Mathlib.GroupTheory.Index
+
+/-!
+# TODO
+
+* Make `α` implicit in `SMulInvariantMeasure`
+* Rename `SMulInvariantMeasure` to `Measure.IsSMulInvariant`
+-/
+
+open Subgroup Set
+open scoped Pointwise
+
+namespace MeasureTheory
+variable {G α : Type*} [Group G] [MeasurableSpace G] [MeasurableMul G] [MeasurableSpace α]
+  {H K : Subgroup G}
+
+@[to_additive]
+instance : MeasurableMul H where
+  measurable_mul_const c := by measurability
+  measurable_const_mul c := sorry -- https://github.com/ImperialCollegeLondon/FLT/issues/276
+
+@[to_additive]
+instance (μ : Measure G) [μ.IsMulLeftInvariant] :
+    (μ.comap Subtype.val : Measure H).IsMulLeftInvariant where
+  map_mul_left_eq_self g := sorry -- https://github.com/ImperialCollegeLondon/FLT/issues/276
+
+@[to_additive]
+instance (μ : Measure G) [μ.IsMulRightInvariant] :
+    (μ.comap Subtype.val : Measure H).IsMulRightInvariant where
+  map_mul_right_eq_self g := sorry -- https://github.com/ImperialCollegeLondon/FLT/issues/276
```

Comment:
Issue #276 talks about a "measurable group". I don't know what that is, but I am surprised that inverse being measurable is not mentioned here. Is there a standard notion of measurable group?

Reply:
I didn't need `MeasurableInv`, but it can easily be stated